include/libvfio-user.h

```diff
@@ -60,11 +60,8 @@ extern "C" {
 
 #define VFU_DMA_REGIONS  0x10
 
-// FIXME: too common a name?
-typedef uint64_t dma_addr_t;
-
 typedef struct {
-    dma_addr_t dma_addr;
+    void *dma_addr;
```

Comment:
do we want to implement the TODO below now maybe?

Reply:
> do we want to implement the TODO below now maybe?

Not sure it's a good idea in the end as we'd have to cast `iovec.iov_base` to `int`. IMO we should drop the FIXME.

Reply:
OK, but I was thinking that was a typo, and it meant dma_addr+length, which *is* like an iov?